src/hotspot/share/opto/divnode.cpp

```diff
@@ -541,6 +541,13 @@ Node *DivINode::Ideal(PhaseGVN *phase, bool can_reshape) {
   // Dividing by MININT does not optimize as a power-of-2 shift.
   if( i == min_jint ) return nullptr;
 
+  // Keep this node as-is for now; we want Value() and
+  // other optimizations checking for this node type to work
```

Comment:
```suggestion
  // Keep this node as-is initially; we want Value() and
  // other optimizations checking for this node type to work.
```
This is a small nit: I find it confusing to talk about "for now" in a method that is called at almost every stage of the compilation. Perhaps "initially" conveys the intention of first letting other optimizations do their magic first a bit better.